tests/test_constants_center_frequencies.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+import pytest
+from numpy import testing as npt
+import pyfar
+
+import pyfar.constants
+
+
+def test_center_frequencies_iec():
+    nominal_octs = [16, 31.5, 63, 125, 250, 500, 1000, 2000, 4000, 8000, 16000]
+    actual_octs = pyfar.constants.fractional_octave_frequencies_nominal(
+        frequency_range=(12, 20e3))
```

Comment:
```suggestion
        frequency_range=(16, 16e3))
```
this should also return the same, shouldn't it?